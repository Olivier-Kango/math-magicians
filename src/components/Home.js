import React from 'react';

const h3 = {
  margin: '60px 0',
};

const Home = () => (
  <div>
    <h3 style={h3}>Welcome to our page!</h3>
    <p>
      In previous projects, I created the core functionality of the app:a calculator component.
      Now I develop a full website with several pages.
    </p>
    <p>
      I create a website consisting of 3 pages: Home, Calculator, and Quote.
      To navigate through those pages, you have to click to those links above.
    </p>
  </div>
);

export default Home;
