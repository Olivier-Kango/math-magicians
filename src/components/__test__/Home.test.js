import { render, screen } from '@testing-library/react';
import Home from '../Home';

describe('Testing the Home page:', () => {
  test('should render', () => {
    render(<Home />);
    const textElement = screen.getByText(/Welcome to our page!/i);
  expect(textElement).toMatchSnapshot();
  });
});