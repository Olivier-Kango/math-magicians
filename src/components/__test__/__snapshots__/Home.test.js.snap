// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Home page: should render 1`] = `
<h3
  style="margin: 60px 0px;"
>
  Welcome to our page!
</h3>
`;
